### ISSUE TITLE
Change target directory for deploys

### ISSUE THREAD

**Issue by Nate-Wilkins** [Syntaf/travis-sphinx#17]
Would it be possible to remove this check?

```python
if sys.argv[-1] == 'deploy':
    print('outdir option not allowed for deploy')
    sys.exit(2)
```

I'm unable to change the target directory on deploy which seems like a bug.

**Comment by Syntaf** [comment]
Good point Nate, but is there any reason why you/others would need to change the output directory?

The reasoning for the unchangeable location is that I branched travis-sphinx originally from my Rust code, which if you aren't familiar with the language, it's internal documentation toolchain will always build to the `target/doc/build` dir.

**Comment by Nate-Wilkins** [comment]
Hmm I guess just more of a convenience. I'm not really using Rust in my project and I guess for more control? Idk up to you :+1:  I can submit a PR if you'd like

**Comment by halkeye** [comment]
I'll throw it out that you can build to a specific target directory, anywhere you want, but you can't deploy anywhere but that hard coded path.

It seems pretty easy and simple to remove that check and doesn't actually hurt anything from providing those two options for deploy and build
